tests/folium_tests.py

```diff
@@ -139,6 +139,38 @@ def test_wms_layer(self):
                                'wms_transparent': 'true'})
         assert map.template_vars['wms_layers'][0] == wms
 
+    def test_add_tile_layer(self):
+        '''Test add_layer URLs'''
+
+        map = folium.Map(location=[44, -73], zoom_start=3)
+        tile_name = "Temperature"
+        tile_url = 'http://gis.srh.noaa.gov/arcgis/services/NDFDTemps/'
+        tile_url += 'MapServer/WMSServer'
```

Comment:
Are any of these options specific to a WMS version?  If so we have to document it!